docs/features/Installation.md

```diff
@@ -37,28 +45,34 @@ The Common Gateway is a Common Ground application built from separate components
 
 ### Installation through Helm charts (recommended method)
 
-[Helm charts](https://artifacthub.io/packages/helm/commonground-gateway/commonground-gateway)) for the Common Gateway are provided through [ArtifactHub](https://artifacthub.io/packages/helm/commonground-gateway/commonground-gateway). Remember that these Helm files only install the gateway and not any attached applications. You will need Helm v3
+Before you continue, make sure you use helm V3 for all the following steps.
+
+And before installing the gateway through Helm charts, make sure you configure cert-manager on your cluster. For more information please visit the [cert-manager documentation](https://cert-manager.io/docs/).
+
+Also make sure to also install the nfs provisioner, see: [ArtifactHub](https://artifacthub.io/packages/helm/kvaps/nfs-server-provisioner). When installing nfs make sure you install with persistance enabled. And make sure to configure at least a persistence size of 1Gi (8Gi or even higher is recommended). The persistence size must be at least as big as the gateway vendor Persistent Volume Claim(s combined) on your cluster.
```

Comment:
More than 1 Gi, because the gateway will try to reserve 1 Gi, which (ironically) does not fit in 1 Gi.